ipmikvm

```diff
@@ -126,22 +129,27 @@ IP=${1:-}; shift
 test -z "$IP" && usage
 
 # Try the aliases/password file.
-DICT="$HOME/.config/ipmikvm/dict"
-if test -s "$DICT"; then
-    # Alias match
-    LINES=$(awk "/^ *[^#]/{if(NF>=4&&(\$1==\"$IP\"||\$2==\"$IP\")){
-        print \$0;exit}}" "$DICT")
-    if test -n "$LINES"; then
-        IP=$(echo "$LINES" | awk '{print $2}')
-        test -z "$USER" && USER=$(echo "$LINES" | awk '{print $3}')
-        test -z "$PASS" && PASS=$(echo "$LINES" | awk '{print $4}')
-    elif test -z "$USER" && test -z "$PASS"; then
-        # No user/pass supplied. Then get all the * * matches.
-        USERS=$(awk '/^ *[^#]/{if(NF>=4&&$1=="*"&&$2=="*"){
-            printf "%s ", $3}}' "$DICT")
-        PASSES=$(awk '/^ *[^#]/{if(NF>=4&&$1=="*"&&$2=="*"){
-            printf "%s ", $4}}' "$DICT")
+if test -e "$DICT"; then
+    TEMPDICT=$(mktemp)
```

Comment:
```
mktemp /dev/shm/ipmikvm.XXXXXX
```
That should write the temp file to memory only and not end up on a (unencrypted) drive.

Reply:
Or, even better: https://github.com/ossobv/vcutil/pull/38#issuecomment-2129530922

Reply:
Done.

Reply:
> Or, even better: [#38 (comment)](https://github.com/ossobv/vcutil/pull/38#issuecomment-2129530922)

ah, will try